sale_date_order_editable/models/sale_order.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 C2i Change 2 improve <soporte@c2i.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    date_order = fields.Datetime(
+        string="Order Date",
+        required=True,
+        readonly=False,
+        index=True,
+        states={"draft": [("readonly", False)], "sent": [("readonly", False)]},
+        copy=False,
+        default=fields.Datetime.now,
+        help="Creation date of draft/sent orders,\nConfirmation date of confirmed orders.",
+    )
```

Comment:
Can you strictly redefine only the field attributes that need change?
That makes code smaller and prevents clash with other modules.

Reply:
@dreispt done